### PR TITLE
Hi, A request for dominic to be available on jsdelivr

### PR DESCRIPTION
**Pull request by bigopon**
Dominic is a library for developer to build up some layout and setup interaction quickly without writing heavy html and CSS to test ideas.

### CODE REVIEW

**Comment by jimaek** [comment]
No version tags and it looks too fresh. I would wait until it becomes more stable before pushing to jsDelivr

**Comment by bigopon** [comment]
Thanks for your response. I thought anything can be published on jsdelivr.

I'll keep pushing some effort towards it and request later.

Wish you a good day

Regards,

**Comment by megawac** [comment]
>  I thought anything can be published on jsdelivr.

Yeah, but for auto-updating we need to know what the version is and what to source it from (i.e. the commit consider a stable release source). You can also just manually upload the project files if they don't put tags on their project.

File: files/dominic/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "bigopon"
+github = "https://github.com/bigopon"
```
Comment (megawac) [inline files/dominic/info.ini]: this url is wrong

File: files/dominic/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+homepage = "http://bigopon.surge.sh"
+description = "Junior web developer"
+mainfile = "dist/dominic.min.js"
```
Comment (megawac) [inline files/dominic/info.ini]: this file should be `dominic.min.js`

File: files/dominic/update.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "dominic",
+  "repo": "bigopon/dominic",
+  "files": {
+    "include": ["dist/dominic.min.js"]
```
Comment (megawac) [inline files/dominic/update.json]: this path doesn't exist.

**Comment by megawac** [comment]
Anyway @bigopon either change the source from `github` to `npm` and the bot can find the tags or start using `git tag` to mark releases. If you're publishing to npm I recomend using `npm version <major|minor|patch>`

**Pull request closed without merging**